src/dispatcher.cpp

```diff
@@ -113,3 +118,15 @@ bool dispatcher::flush()
     } );
     return invoked;
 }
+
+bool dispatcher::_wait_for_start( int timeout_ms )
+{
+    // If the dispatcher is not started wait for a start event, if not such event within given timeout do nothing.
+    // If during the wait the thread destructor is called (_is_aliva = false) do nothing as well.
+    //
```

Comment:
Please repeat the return value meaning as you wrote in the header

Reply:
👍